build.gradle.kts

```diff
@@ -1,7 +1,7 @@
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
 
 plugins {
-  id("uk.gov.justice.hmpps.gradle-spring-boot") version "5.1.4-beta-2"
+  id("uk.gov.justice.hmpps.gradle-spring-boot") version "5.1.4"
   kotlin("plugin.spring") version "1.8.10"
```

Comment:
shouldn't really go to `5.14` without updating kotlin version to `1.8.21`

Reply:
@petergphillips updated